src/main/kotlin/no/nav/sbl/sosialhjelp_mock_alt/otherEndpoints/frontend/model/FrontendPersonalia.kt

```diff
@@ -245,6 +268,27 @@ data class FrontendUtbetalingFraNav(
                 utbetaling.komponenter.first().type ?: ""
             )
         }
+
+        // TODO: bytt ut metode over med denne når vi har skrudd over til utbetalData. Pt ubrukt.
+        fun mapUtbetalDataDtoToFrontendUtbelingFraNav(utbetalData: UtbetalDataDto): FrontendUtbetalingFraNav {
+
+            // TODO  en del verdier her som er én verdi kommer fra lister. Velger nå første i liste, men hvordan skal logikken her henge sammen?
```

Comment:
Disse TODOene løses senere?

Reply:
Ja, når vi er over på ny tjeneste. Hadde ikke noe bedre sted å putte det for å ikke glemme det. Nå vi skulle hatt det boardet vårt som IKKE er Jira ;-)